lib/model.js

```diff
@@ -605,7 +605,7 @@ _.assign(derived, {
         });
       } else if (this.authentication === 'LDAP') {
         req.auth = format('%s:%s',
-          encodeURIComponent(this.ldap_username),
+          this.ldap_username,
```

Comment:
@rueckstiess As @pzrq pointed out in 3e8397d15ece01739d589c576cb4ab09d9d3f379,
the second encoding was taking place when we call node.js's built [`url.format(data)`](https://github.com/mongodb-js/connection-model/blob/3e8397d15ece01739d589c576cb4ab09d9d3f379/lib/model.js#L627) method.